package.json

```diff
@@ -1,66 +1,128 @@
 {
-	"name": "is-string",
-	"version": "1.0.4",
-	"author": "Jordan Harband",
-	"description": "Is this value a JS String object or primitive? This module works cross-realm/iframe, and despite ES6 @@toStringTag.",
-	"license": "MIT",
-	"main": "index.js",
-	"scripts": {
-		"test": "npm run lint && node --harmony --es-staging test.js && npm run security",
-		"coverage": "covert test.js",
-		"coverage-quiet": "covert test.js --quiet",
-		"lint": "npm run jscs && npm run eslint",
-		"jscs": "jscs test.js *.js",
-		"eslint": "eslint test.js *.js",
-		"eccheck": "editorconfig-tools check *.js **/*.js > /dev/null",
-		"security": "nsp package"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git://github.com/ljharb/is-string.git"
-	},
-	"keywords": [
-		"String",
-		"string",
-		"ES6",
-		"toStringTag",
-		"@@toStringTag",
-		"String object"
-	],
-	"dependencies": {},
-	"devDependencies": {
-		"foreach": "^2.0.5",
-		"is": "^3.1.0",
-		"tape": "^4.2.0",
-		"indexof": "^0.0.1",
-		"covert": "^1.1.0",
-		"jscs": "^2.1.1",
-		"editorconfig-tools": "^0.1.1",
-		"nsp": "^1.1.0",
-		"eslint": "^1.5.1",
-		"@ljharb/eslint-config": "^1.2.0",
-		"semver": "^5.0.3"
-	},
-	"testling": {
-		"files": "test.js",
-		"browsers": [
-			"iexplore/6.0..latest",
-			"firefox/3.0..6.0",
-			"firefox/15.0..latest",
-			"firefox/nightly",
-			"chrome/4.0..10.0",
-			"chrome/20.0..latest",
-			"chrome/canary",
-			"opera/10.0..latest",
-			"opera/next",
-			"safari/4.0..latest",
-			"ipad/6.0..latest",
-			"iphone/6.0..latest",
-			"android-browser/4.2"
-		]
-	},
-	"engines": {
-		"node": ">= 0.4"
-	}
+  "_args": [
+    [
+      {
+        "raw": "is-string",
```

Comment:
please revert all these changes to package.json - you need to `git clone`, not make changes from the npm-installed version of the package.